great_expectations/dataset/base.py

```diff
@@ -664,101 +663,101 @@ def validate(self, expectations_config=None, catch_exceptions=True, output_forma
 
     ##### Output generation #####
     def _format_column_map_output(self,
-        output_format, success,
-        element_count,
-        nonnull_values, nonnull_count,
-        boolean_mapped_success_values, success_count,
-        exception_list, exception_index_list
-    ):
+                                  result_format, success,
```

Comment:
Are *you* the one who's been adding all these deep indents? Makes it so much harder to stay close to the 79-character limit from PEP 8.

Reply:
To quote PEP8 (https://www.python.org/dev/peps/pep-0008/#code-lay-out): 

```
Code lay-out
Indentation
Use 4 spaces per indentation level.
```

😄 😄

Reply:
We should look up the standard for this...